src/repos/NotificationRepo.ts

```diff
@@ -114,10 +114,10 @@ const notifyNewMagazines = async (magazineIDs: string[]): Promise<void> => {
     });
   });
 };
-
 /**
  * Send notifications for new flyers posted by organizations.
  */
+
 const notifyNewFlyers = async (flyerIDs: string[]): Promise<void> => {
   flyerIDs.forEach(async (f) => {
     const flyer = await FlyerRepo.getFlyerByID(f); // eslint-disable-line
```

Comment:
When you use `eslint-disable-line`, can you write a comment explaining why you need it? I believe this tends to be best practice since using comments like this to work around checkers is something you want to avoid generally.